### PR TITLE
add cloud-platform scope in the test to reclaim the ai platform sample models

### PR DESCRIPTION
**Pull request by gaoning777**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2355)

### CODE REVIEW

File: sdk/python/kfp/_client.py [hunk 1]
```diff
@@ -358,7 +358,7 @@ def __init__(self, client, run_info):
 
       def wait_for_run_completion(self, timeout=None):
         timeout = timeout or datetime.datetime.max - datetime.datetime.min
-        return self._client.wait_for_run_completion(timeout)
+        return self._client.wait_for_run_completion(self.run_id, timeout)
```
Comment (Ark-kun) [inline sdk/python/kfp/_client.py]: Good catch!

File: test/deploy-cluster.sh [hunk 2]
```diff
@@ -48,7 +48,7 @@ else
   SHOULD_CLEANUP_CLUSTER=true
   # "storage-rw" is needed to allow VMs to push to gcr.io
   # reference: https://cloud.google.com/compute/docs/access/service-accounts#accesscopesiam
-  SCOPE_ARG="--scopes=storage-rw"
+  SCOPE_ARG="--scopes=storage-rw,cloud-platform"
```
Comment (numerology) [inline test/deploy-cluster.sh]: I think this is the line solving the permission scope issue, right?

**Pull request merged**